dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/internal/services/OpenAPIVersionConfig.java

```diff
@@ -0,0 +1,16 @@
+package io.openliberty.microprofile.openapi20.internal.services;
```

Comment:
Missing copyright header

Reply:
Bah, I must have done this one before I noticed the project wasn't configured to add them automatically. Fixed.